addons/mail/models/mail_thread.py

```diff
@@ -1653,7 +1653,7 @@ def _message_get_suggested_recipients(self):
         """ Returns suggested recipients for ids. Those are a list of
         tuple (partner_id, partner_name, reason), to be managed by Chatter. """
         result = dict((res_id, []) for res_id in self.ids)
-        if 'user_id' in self._fields:
+        if 'user_id' in self._fields and self._fields['user_id'].comodel_name == 'res.users':
```

Comment:
I think this is enough sir :raised_eyebrow:  @tde-banana-odoo

Reply:
`comodel_name` is only available for Relational fields. Then it will still crash for the use case described in your PR description (for Char Fields), right ? IMO, you should also check the type of fields ( == 'many2one').

Reply:
I am aware of your case and test on my custom module, and the `comodel_name` give me none not crash at all (i have `user_id` as a char field ). 
Ok i will check `many2one` to be sure because this method expect many2one not x2many, if we have something like `user_id = fields.x2many` then it will crash when go to the ` obj._message_add_suggested_recipient(result, partner=obj.user_id.partner_id, reason=self._fields['user_id'].string)`

Reply:
Yep, you are right `comodel_name` is defined in `Field` directly. But indeed x2many shouldn't take in account.

Reply:
I have updated please review sir